zio-http/src/main/scala/zio/http/netty/Versions.scala

```diff
@@ -0,0 +1,21 @@
+package zio.http.netty
+
+import io.netty.handler.codec.http.HttpVersion
+import zio._
+import zio.http.model.Version
+
+object Versions {
+  import Version._
+  def make(version: HttpVersion)(implicit unsafe: Unsafe, trace: Trace): Version =
+    version match {
+      case HttpVersion.HTTP_1_0 => Http_1_0
+      case HttpVersion.HTTP_1_1 => Http_1_1
+      case _                    => throw new IllegalArgumentException(s"Unsupported HTTP version: $version")
+    }
+
+  def make(version: Version): HttpVersion = version match {
```

Comment:
This can throw exceptions.

Reply:
Or maybe not.

Reply:
`Versions` removed and all code now inlined into `SeverInboundHandler`.